pkg/actions/cluster/owned.go

```diff
@@ -54,3 +60,64 @@ func (c *OwnedCluster) Upgrade(dryRun bool) error {
 	cmdutils.LogPlanModeWarning(dryRun && (stackUpdateRequired || versionUpdateRequired))
 	return nil
 }
+
+func (c *OwnedCluster) Delete(waitTimeout time.Duration, wait bool) error {
+	var (
+		clientSet kubernetes.Interface
+		oidc      *iamoidc.OpenIDConnectManager
+	)
+
+	clusterOperable, _ := c.ctl.CanOperate(c.cfg)
```

Comment:
similar to above, does this error matter? (i appreciate you moved this from somewhere else)

Reply:
Same as https://github.com/weaveworks/eksctl/pull/2969#discussion_r552524997, added a log message